pkg/k8shandler/reconciler.go

```diff
@@ -35,20 +34,27 @@ func Reconcile(requestCluster *logging.ClusterLogging, requestClient client.Clie
 		clusterLoggingRequest.ForwarderSpec = forwarder.Spec
 	}
 
+	clusterLoggingRequest.Cluster.Status.Conditions.SetCondition(logging.NewCondition("ConditionUnknown",
```

Comment:
@alanconway thoughts?  This is a condition that would constantly flip.  Do we want to do this every cycle.  I'm betting it may be difficult for a user to catch

Reply:
+1 drop this condition.
Status is not the right place for transient "in progress" messages, it is for long-lasting conditions that won't change unless there is an external change: either a change to the API or a change in the state of some operand.

Reply:
ok